R/parameters.R

```diff
@@ -1,28 +1,39 @@
 #' @title Get parameters from SQUIRE model
 #'
 #' @param pop population. See [squire::get_population]
-#' @param dt the timestep (days) for the simulation
+#' @param R0 R0
 #' @param time_period the number timesteps to run the simulation for
-#' @param ... Other parameters for [squire::parameters_explicit_SEEIR] 
+#' @param tt_contact_matrix tt_contact_matrix
```

Comment:
tt_contact_matrix and contact_matrix_set are dealt with in paramters_explicit_SEEIR. It feels redundant to deal with them here too.

Can we leave it how it was?

Reply:
it didn't work that's why I added them  - talk offline